csfieldguide/static/interactives/pixel-viewer/js/pixel-viewer.js

```diff
@@ -150,8 +150,9 @@ function setUpMode(){
   }
 }
 
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/58444672-66487400-814d-11e9-8dca-250d32cdca28.png)
Is this correct, for median blur on rgb?

Reply:
Yeah I think so. It looks better if you use a better type of blur. That's how it currently looks on the live site anyway